App.js

```diff
@@ -27,32 +29,49 @@ const App = () => {
     const parsedRes = await res.json();
     setData(parsedRes);
   };
+
   const [data, setData] = useState();
 
   useEffect(() => {
     fetchData();
   }, []);
 
-  console.log(data);
+   //Function used by flatlist that indicates how the item is going to be seen
+  const renderItem =  ( {item} ) => (
+    <Fact key={item._id} fact={item} />
+  )
+  
   return (
     <>
       <StatusBar barStyle="dark-content" />
       <SafeAreaView>
-        <ScrollView
-          contentInsetAdjustmentBehavior="automatic"
-          style={styles.scrollView}>
-          <Text>Cat facts</Text>
-          {data && data.all.map((item) => <Fact key={item._id} fact={item} />)}
-        </ScrollView>
+          <Text style={styles.maintext} >Cat facts</Text>
+          { data &&     
```

Comment:
here it should be `{data &&` the spaces we leave inside curly braces are usually only for imports, inside object definitions or object destructures.

Having eslint and prettier configured this will fix itself but it's good to point out now.